### PR TITLE
InOb Repeat

### PR DESCRIPTION
**Pull request by jonkeller**
Renames "reset" to "repeat".
Subsequent work in a separate PR will tweak the logic of repeat.

### CODE REVIEW

File: extensions/amp-analytics/0.1/visibility-model.js [hunk 1]
```diff
@@ -200,23 +200,24 @@ export class VisibilityModel {
    * Function that visibilityManager can used to dispose model or reset model
    */
   disposeOrReset() {
```
Comment (bradfrizzell) [inline extensions/amp-analytics/0.1/visibility-model.js]: should this be disposeOrRepeat?

Comment (jonkeller) [inline extensions/amp-analytics/0.1/visibility-model.js]: ^^ This one intentionally left as reset.

File: extensions/amp-analytics/0.1/visibility-model.js [hunk 2]
```diff
@@ -200,23 +200,24 @@ export class VisibilityModel {
    * Function that visibilityManager can used to dispose model or reset model
```
Comment (bradfrizzell) [inline extensions/amp-analytics/0.1/visibility-model.js]: "or repeat model"

File: extensions/amp-analytics/0.1/visibility-model.js (hunk 1 above)
Comment (jonkeller) [reply extensions/amp-analytics/0.1/visibility-model.js]: I think no but not 100% sure

File: extensions/amp-analytics/0.1/visibility-model.js (hunk 2 above)
Comment (jonkeller) [reply extensions/amp-analytics/0.1/visibility-model.js]: Intentionally did not change this one, but not 100% sure that was correct

**Comment by jonkeller** [comment]
@lannka Correct, that is the plan. Thanks!

**Pull request merged**